Example.ipynb

```diff
@@ -20,74 +20,7 @@
    "outputs": [
     {
      "data": {
-      "image/svg+xml": [
```

Comment:
I reran the notebook. Most of the changes (in terms of the number of lines) are due to Jupyter's compactified notebook representation compared to 5 years ago.

Some of the figures are different due to python library refactorings.

Additional cells at the bottom added to demonstrate the new functionality.